src/software/backend/backend.h

```diff
@@ -17,10 +17,21 @@
 class Backend : public Subject<SensorMsg>,
                 public Subject<World>,
```

Comment:
should this be removed?

Reply:
no, because we haven't changed the observer connections in AI yet so removing this would break that